qutip/solver/brmesolve.py

```diff
@@ -198,17 +199,22 @@ def brmesolve(
             a_op = QobjEvo(a_op, args=args, tlist=tlist)
         if isinstance(spectra, str):
             new_a_ops.append(
-                (a_op, coefficient(spectra, args={**args, 'w':0})))
+                (a_op, coefficient(spectra, args={**args, 'w': 0})))
         elif isinstance(spectra, InterCoefficient):
             new_a_ops.append((a_op, SpectraCoefficient(spectra)))
         elif isinstance(spectra, Coefficient):
             new_a_ops.append((a_op, spectra))
+        elif isinstance(spectra, BosonicEnvironment):
+            spec = SpectraCoefficient(
+                coefficient(lambda w: spectra.power_spectrum(w))
```

Comment:
```suggestion
                coefficient(spectra.power_spectrum)
```
Is there something that requires the use of a lambda here?

Reply:
I don't think so, changed